main/http_server_cb.c

```diff
@@ -902,6 +902,11 @@ http_server_cb_on_post_ruuvi(const char *p_body)
         {
             ethernet_update_ip();
         }
+        const force_start_wifi_hotspot_t force_start_wifi_hotspot = settings_read_flag_force_start_wifi_hotspot();
+        if (FORCE_START_WIFI_HOTSPOT_PERMANENT == force_start_wifi_hotspot)
+        {
+            settings_write_flag_force_start_wifi_hotspot(FORCE_START_WIFI_HOTSPOT_DISABLED);
```

Comment:
Is it intentional to write `FORCE_START_WIFI_HOTSPOT_DISABLED` to settings inside 
```
if (FORCE_START_WIFI_HOTSPOT_PERMANENT == force_start_wifi_hotspot)
{
}
```
?

Reply:
I added a comment:

```
            /* A permanent start-up of the Wi-Fi hotspot should be performed after each reboot
             * only until a new non-default configuration is saved */

```